docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -95,27 +104,66 @@ Before using this method, you must have set up a https://learn.microsoft.com/en-
 . Go to your Azure subscription list and select the subscription you want to monitor with CSPM.
 . Go to **Access control (IAM)** and select **Add Role Assignment**. 
 . Select the `Reader` function role, assign access to **User, group, or service principal**, and select your new app.
+. Return to the **Add CSPM** page in {kib}. 
```

Comment:
Would it be helpful to add before the step `Return to the **Add CSPM** page in {kib}.` , another step about creating the secret in App registration? 
e.g. something like:
. Go to App Registration you created for the agent, in tab  **Certificate & Secrets** and then **Client secrets** tab, and create and copy a new secret for the elastic agent.

Reply:
Maybe I'm missing something, but right now line 103 has something very similar to what you're suggesting here. Do users need to make a second secret at this point or is the existing instruction enough?

Reply:
Yes, you are right,  my mistake. This is the step.